docs/code-reference/META.md

```diff
@@ -573,7 +573,7 @@ This file tracks code elements that need documentation.
 
 ## rest-api/class-acf-rest-api.php
 
-### Classs
+### Classes
```

Comment:
We to update this in the `docs/bin/generate-parsed-md.php` file, line ~544 to:
```
				if ( 'class' === $type ) {
					$type = 'classes';
				}
```

Reply:
👍

Reply:
ah! i understand now, it's just pluralising the names

Reply:
See: https://github.com/WordPress/secure-custom-fields/pull/77/commits/de86a3f0f111228ea2d578276d6f5ab95d03aa50